src/index.js

```diff
@@ -77,6 +77,11 @@ const Reliable = or(
   UTP
 )
 
+const Stardust = or(
+  and(Reliable, base('p2p-stardust'), base('ipfs')),
```

Comment:
Just wanted to note: Unlike ws-star, stardust can run over _any_ transport thus removing the requirement for ws-star (but don't even think about running a stardust server over webrtc-star ;) )

Reply:
Added as comment into the code